### PR TITLE
string-machine: init at unstable-2020-01-20

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/audio/string-machine/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, boost, cairo, lv2, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "string-machine";
+  version = "unstable-12-11-2019";
```
Comment (jonringer) [inline pkgs/applications/audio/string-machine/default.nix]:
```suggestion
  version = "unstable-2019-11-12";
```

Comment (magnetophon) [reply pkgs/applications/audio/string-machine/default.nix]: OK, done.

**Comment by jonringer** [comment]
please update the commit message:
```
string-machine: init at unstable-2019-11-12
```

**Comment by magnetophon** [comment]
@jonringer Sorry, slipped trough the cracks!

File: pkgs/applications/audio/string-machine/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  buildInputs = [
+    boost cairo lv2
+  ];
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/jpcima/string-machine;
```
Comment (jonringer) [inline pkgs/applications/audio/string-machine/default.nix]:
```suggestion
    homepage = "https://github.com/jpcima/string-machine";
```

Comment (magnetophon) [reply pkgs/applications/audio/string-machine/default.nix]: Thanks, all done.

**Review by jonringer — APPROVED** [review]
diff LGTM

```
[3 built, 1 copied (1.1 MiB), 0.2 MiB DL]
https://github.com/NixOS/nixpkgs/pull/74041
1 package built:
string-machine
```

File: pkgs/applications/audio/string-machine/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/jpcima/string-machine";
+    description = "Digital model of electronic string ensemble instrument";
+    maintainers = [ maintainers.magnetophon ];
+    platforms = platforms.linux;
```
Comment (jonringer) [inline pkgs/applications/audio/string-machine/default.nix]: uses x86 specific extensions, please do one of the following
```suggestion
    platforms = intersectLists platforms.linux platforms.x86;
```
or
```suggestion
    platforms = [ "i686-linux" "x86_64-linux" ];
```

**Review by jonringer — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/string-machine/default.nix (hunk 3 above)
Comment (magnetophon) [reply pkgs/applications/audio/string-machine/default.nix]: Done!
Also upped the version.

**Review by jonringer — APPROVED** [review]
LGTM

```
[3 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/74041
1 package built:
string-machine
```

**Pull request merged**